content/academy/apify_platform/publishing_actors_on_apify_store/monetizing_your_actor.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Monetizing your actor
+description: Learn how you can use Apify to monetize your web scraping and automation projects.
+menuWeight: 2
+paths:
+  - apify-platform/publishing-actors-on-apify-store/monetizing-your-actor
+---
+
+# [](#monetizing-your-actor) Monetizing your actor
+
+When publishing your actor on the Apify platform you have the choice to make it a **Paid actor** and get paid by the users benefiting from your tool.
+
+![Paid actor example](https://i.postimg.cc/k5VHybtm/scraper-example.png)
```

Comment:
All images should be saved in the repo and minified using tinyPNG. See other lessons for examples how it's done.